libctru/include/3ds/applets/browser.h

```diff
@@ -0,0 +1,12 @@
+/**
+ * @file browser.h
+ * @brief Browser Applet
+ */
+
+#pragma once
+
+/**
+ * @brief Open the browser at a url
+ * @param url The url to open at, max-length 1024 characters including the trailing zero. If invalid or null, just opens the browser.
+ */
+void browserOpenUrl(char* url);
```

Comment:
```suggestion
void browserOpenUrl(const char* url);
```

Reply:
argh, somehow thought she had committed this